node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -1236,11 +1311,11 @@ where
 
 		entry.insert(collation_event);
 	} else {
-		tracing::error!(
+		tracing::debug!(
 			target: LOG_TARGET,
 			?relay_parent,
 			candidate = ?candidate_receipt.hash(),
-			"Trying to insert a pending candidate failed, because there is already one!",
+			"Trying to insert a pending candidate failed, because there is already one.",
```

Comment:
As this now is legal, we probably can remove this

Reply:
I made it debug, now even lowered it to trace.